### ISSUE TITLE
Breadcrumbs as headings example: illogical headings

### ISSUE THREAD

**Issue by jlukosky** [OfficeDev/office-ui-fabric-react#10750]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

In the [Breadcrumbs as headings](https://developer.microsoft.com/en-us/fabric/#/controls/web/breadcrumb) example the order of the headings goes from `h4` to `h3` to `h2`: 

- `h4`: Files
- `h3`: This is folder 1
- `h2`: This is folder 2

This is not a logical order. If these were each headings (and they shouldn't be) Files would be the most important with "This is folder 1" being the first item under it and "This is folder 2" the first item under that. Therefore the order would be:

- `h2`: Files
- `h3`: This is folder 1
- `h4`: This is folder 2

But as I said, it doesn't make sense for each of these to be headings. This is a navigation. It might make sense for the _last_ item to be a heading but not the others.

I think this was coded this way because someone wanted to not have to have a heading on their page and decided to jam it in the breadcrumb. It's key to recognize what content is falling under each heading. 

#### Actual behavior:

Headings within the breadcrumb nav do not logically act as headings. They are broken in a way that _might_ pass automated checking.

Observed heading levels
- `h4`: Files (What is this a heading of?)
- `h3`: This is folder 1 (What is this a heading of?)
- `h2`: This is folder 2 

#### Expected behavior:

- Files (Should not be a heading)
- This is folder 1 (Should not be a heading)
- `h2`: This is folder 2 (Probably shouldn't be a heading but that will depend on context so let's leave it to the author)

**Comment by ecraig12345** [comment]
@maxwellred Since you added that example, could you provide more background on why you did it that way? I'm also curious what the actual scenario was that motivated adding support for heading breadcrumbs (and whether we could make the example more like the real scenario).

**Comment by ecraig12345** [comment]
Breadcrumbs as headings were added in PR #10445 in response to issue #10286. The actual scenario was to allow making the current item a heading. So I'll update the current example to use a heading only for the current item (probably an `h4` in this case to minimize messing with the structure of the page).
